bcda/models/models.go

```diff
@@ -166,20 +166,29 @@ type ACO struct {
 	ACOBeneficiaries []*ACOBeneficiary
 }
 
-func (aco *ACO) GetBeneficiaryIDs() (beneficiaryIDs []string, err error) {
+func (aco *ACO) GetBeneficiaryIDs() (cclfBeneficiaryIDs []string, err error) {
+	if aco.CMSID == nil {
+		log.Errorf("No CMSID set for ACO: %s", aco.UUID)
+		return cclfBeneficiaryIDs, fmt.Errorf("no CMS ID set for this ACO")
+	}
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
+	var cclfFile CCLFFile
+	// should I put a filter here to make sure it isn't too old?
+	if db.Debug().Where("aco_cms_id = ? and cclf_num = 8", aco.CMSID).Order("timestamp desc").First(&cclfFile).RecordNotFound() {
```

Comment:
Looks like `db.Debug()` might have been left in by accident.  Probably don't want that in production code.

Reply:
fixed in incoming commit